qiskit_nature/algorithms/excited_states_solvers/qeom.py

```diff
@@ -68,21 +68,17 @@ def excitations(self, excitations: Union[str, List[List[int]]]) -> None:
                                  '(singles and doubles)')
         self._excitations = excitations
 
-    def solve(self, driver: BaseDriver,
-              aux_operators: Optional[Union[List[FermionicOperator],
-                                            List[BosonicOperator]]] = None
+    def solve(self, problem: BaseProblem,
+              aux_operators: Optional[List[SecondQuantizedOp]] = None
```

Comment:
Do we need to address also the case where PauliSumOp is provided as an auxiliary operator? Maybe also @woodsp-ibm  has an opinion for this.

Reply:
That's what we also discussed this morning. See also this thread: https://github.com/Qiskit/qiskit-nature/pull/101#discussion_r603848838